pytest/tests/mocknet/README.md

```diff
@@ -1,12 +1,18 @@
 Mirror transactions from a given network into a custom mocktest network and add load
 
-1. Setup a custom mocknet network following the README in the `provisioning/terraform/network/mocknet/mirror/` directory of the [near-ops](https://github.com/near/near-ops) repo.
-- you'll need the `unique_id`, `chain_id`, and `start_height` from this setup when running the mirror.py test script in 2.
+1. Setup a custom mocknet network following the [README](https://github.com/Near-One/infra-ops/blob/main/provisioning/terraform/infra/network/mocknet/mirror/README) in the `provisioning/terraform/infra/network/mocknet/mirror/` directory of the [Near-One/infra-ops repository](https://github.com/Near-One/infra-ops).
+    - An example setup command should look like the following: `terraform apply -var="unique_id=stateless" -var="chain_id=mainnet" -var="start_height=116991260" -var="size=small"`
+
+    - Use the same values of `unique_id`, `chain_id`, and `start_height` from this setup when running the mirror.py commands below. 
+
 2. Run `python3 tests/mocknet/mirror.py --chain-id {chain_id} --start-height {start_height} --unique-id {unique_id} init-neard-runner`, replacing the `{}`s with appropriate values from the `nearcore/pytest` directory. This starts a helper program on each node that will be in charge of the test state and neard process.
```

Comment:
I wonder if we can derive chain id and start height from unique id.

Reply:
Not sure, I will check out with others. But right we should be able to figure out or maybe hard-code the mainnet and stateless somewhere.

Reply:
hmm ya good point I bet we can actually... because most of the time theres only going to be one network with a given start height. so we can just look thru all the instances with that unique id and if theres multiple start heights then quit with an error saying it's underspecified, but most of the time just allow only `--unique-id`